auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -522,6 +538,12 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
     this.$refs.errorDialog.open()
   }
 
+  showBusinessAlreadyAdded (event) {
+    this.dialogTitle = 'Business Already Added'
+    this.dialogText = `The business ${event.name} with the businss number ${event.identifier} is already in your Business Registry List.`
+    this.$refs.errorDialog.open()
+  }
```

Comment:
Why is this needed, since already-existing businesses should appear as "Added" in the search results and not be clickable?